index.md

```diff
@@ -2,6 +2,7 @@
 layout: HubPage
 hide_bc: true
 title: .NET Documentation
+description: Learn how to use .NET to create a variety of apps using the language, platforms and tools of your choice.
```

Comment:
Learn how to use .NET to create a variety of applications on any platform using C#, Visual Basic, and F#.